test/utils.js

```diff
@@ -172,6 +173,28 @@ export const readFee = (amount, feeStr) => {
   }
 };
 
+export const getTerminalCols = () => {
+  let length = 40;
+  if (process.platform === 'win32') {
+    return length;
+  }
+  try {
+    length = parseInt(execSync('tput cols').toString());
+  } catch {
+    return length;
+  }
+};
+
+export const separator = length => {
+  const terminalCols = length || getTerminalCols();
+
+  let separator = '';
+  for (let i = 0; i < terminalCols; i++) {
+    separator += '-';
+  }
+  console.log(`\n${separator}\n`);
```

Comment:
perhaps remove this console log as well...

Reply:
well no, that's the whole point of this helper function: make it easier to debug in the console by spacing out and separating sections.

Reply:
I see, makes sense! missed that